packages/strapi-plugin-content-manager/services/editing-lock.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const _ = require('lodash');
+const { webhook: webhookUtils } = require('strapi-utils');
+const { TTL, LOCK_PREFIX } = require('./constants');
+
+const { ENTRY_UPDATE, ENTRY_CREATE } = webhookUtils.webhookEvents;
+
+const getLockKey = (model, entryId) => {
+  const { kind } = strapi.getModel(model);
+  return kind === 'singleType' ? `edit:${model}` : `edit:${model}:${entryId}`;
+};
+
+const acceptedMetadata = ['lastActivityDate'];
+
+const getLock = async ({ model, entityId }) => {
+  const lockService = strapi.lockService({ prefix: LOCK_PREFIX });
+  const key = getLockKey(model, entityId);
+  const lockResult = await lockService.get(key);
+  return {
```

Comment:
return lockService.get(key) ?

Reply:
I would prefer not to avoid leaking data in the future. What do you think ?